compiler/qsc_codegen/src/qir.rs

```diff
@@ -136,6 +136,29 @@ impl ToQir<String> for rir::Operand {
     }
 }
 
+impl ToQir<String> for rir::FcmpConditionCode {
+    fn to_qir(&self, _program: &rir::Program) -> String {
+        match self {
+            rir::FcmpConditionCode::False => "false".to_string(),
+            rir::FcmpConditionCode::OrderedAndEqual => "oeq".to_string(),
+            rir::FcmpConditionCode::OrderedAndGreaterThan => "ogt".to_string(),
+            rir::FcmpConditionCode::OrderedAndGreaterThanOrEqual => "oge".to_string(),
+            rir::FcmpConditionCode::OrderedAndLessThan => "olt".to_string(),
+            rir::FcmpConditionCode::OrderedAndLessThanOrEqual => "ole".to_string(),
+            rir::FcmpConditionCode::OrderedAndNotEqual => "one".to_string(),
+            rir::FcmpConditionCode::Ordered => "ord".to_string(),
+            rir::FcmpConditionCode::UnorderedOrEqual => "ueq".to_string(),
+            rir::FcmpConditionCode::UnorderedOrGreaterThan => "ugt".to_string(),
+            rir::FcmpConditionCode::UnorderedOrGreaterThanOrEqual => "uge".to_string(),
+            rir::FcmpConditionCode::UnorderedOrLessThan => "ult".to_string(),
+            rir::FcmpConditionCode::UnorderedOrLessThanOrEqual => "ule".to_string(),
+            rir::FcmpConditionCode::UnorderedOrNotEqual => "une".to_string(),
+            rir::FcmpConditionCode::Unordered => "uno".to_string(),
+            rir::FcmpConditionCode::True => "true".to_string(),
```

Comment:
Very minor nit: But why is `True` last? Seems it should be adjacent to `False`, or alphabetical (i.e. before the `Unordered`s).

Reply:
That is the order in which they are defined in the spec, so I kept it in that order.